kmp.cpp

```diff
@@ -0,0 +1,53 @@
+//Code by:- Devansh Sharma
+
```

Comment:
It would be nice if you also add you GitHub username in the format: @author [username].
This would add help others find your contributions better.